modules/ROOT/pages/twilio-connector.adoc

```diff
@@ -256,4 +258,4 @@ For background information on pooling, see xref:tuning-performance.adoc[Tuning P
 == About Other Resources
 
 * Access the xref:release-notes::connector/twilio-connector-release-notes.adoc[Twilio Connector Release Notes].
-* Visit Twilio's official https://www.twilio.com/docs/api/rest[REST API Reference].
\ No newline at end of file
+* Visit Twilio's official https://www.twilio.com/docs/api/rest[REST API Reference].
```

Comment:
I think we discourage 3rd party URLs, but I'm not gonna say to remove it in this case.

Reply:
It does make sense to minimize 3d party URLs. I think this one is necessary, though.